### PR TITLE
[Let's Encrypt.org] New ruleset

### PR DESCRIPTION
**Pull request by rugk**

### CODE REVIEW

**Comment by reedy** [comment]
Any reason we couldn't rename it (to save some history) and then just update the file?

File: src/chrome/content/rules/LetsEncrypt.org.xml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        and the cross-signature is not in place too. However this will be changed later.
+-->
+<ruleset name="Let's Encrypt">
+    <target host="letsencrypt.org" />
+    <target host="www.letsencrypt.org" />
+    <target host="helloworld.letsencrypt.org" />
+    <target host="community.letsencrypt.org" />
+
+    <!--
+        testing URl which does not redirect to HTTPS by default, see
```
Comment (reedy) [inline src/chrome/content/rules/LetsEncrypt.org.xml]: mixed case on URL?

**Comment by rugk** [comment]
Okay, renamed.

File: src/chrome/content/rules/LetsEncrypt.org.xml (hunk 1 above)
Comment (rugk) [inline src/chrome/content/rules/LetsEncrypt.org.xml]: done

**Comment by fuglede** [comment]
Seems to me that adding helloworld.letsencrypt.org partially defeats the purpose of that page and might lead to possible confusion. I'd suggest leaving that out until the CA is live.

**Comment by fuglede** [comment]
Regarding the securecookie, I wasn't actually able to find one that was not already secured. Can you confirm that those exist? Otherwise there's not much point to that one. (Not that it hurts -- #2907 is relevant here.) As a minor note, <code>www.</code> was probably intended to be <code>www\\.</code> (and similarly for <code>community.</code>).

Finally, the included test URL does in fact redirect to HTTPS (now?);

    $ curl -is http://letsencrypt.org/repository | grep "Location"
    Location: https://letsencrypt.org/repository

**Comment by rugk** [comment]
> Can you confirm that those exist?

No, just added it like it is. As you said it does not hurt.

> As a minor note, www. was probably intended to be www\. (and similarly for community.).

Indeed. I'll change that.

> Finally, the included test URL does in fact redirect to HTTPS (now?);

Yes it seems like it is.

**Comment by rugk** [comment]
So everything was fixed? Does someone wants to merge it?

**Comment by fuglede** [comment]
I would hold that this still breaks the intended behaviour of http://helloworld.letsencrypt.org.

**Comment by rugk** [comment]
Okay, okay. I commented `helloworld.letsencrypt.org` out.

**Comment by fuglede** [comment]
Also, the comment on /repository is still inaccurate.

**Comment by rugk** [comment]
`helloworld.letsencrypt.org` is [now trusted](https://letsencrypt.org/2015/10/19/lets-encrypt-is-trusted.html) by all major browsers, so should I possibly add it back to this ruleset?

**Comment by rugk** [comment]
So what about this PR?

**Comment by fuglede** [comment]
👍

**Pull request merged**